doc/contributing/contributing-to-documentation.chapter.md

```diff
@@ -2,26 +2,13 @@
 
 The sources of the Nixpkgs manual are in the [doc](https://github.com/NixOS/nixpkgs/tree/master/doc) subdirectory of the Nixpkgs repository. The manual is still partially written in DocBook but it is progressively being converted to [Markdown](#sec-contributing-markup).
 
-You can quickly check your edits with `make`:
+You can check your edits with `nix-build`:
 
 ```ShellSession
-$ cd /path/to/nixpkgs/doc
-$ nix-shell
-[nix-shell]$ make
+[nixpkgs]$ nix-build ./doc
```

Comment:
The square bracket prompt is nix-shell thing.

Reply:
I think it's valid to use for working directories, considering the NixOS default prompt of

```
[user@machine:workdir]$ nix-shell 
```

Reply:
We usually omit the default prompt unless it is particularly relevant.

Reply:
`nix-build ./doc` wouldn't work where the file is located, so there's some ambiguity, but none that isn't easily resolved by inference. I guess that's where we draw the line then?

Reply:
I would say only use detailed prompt when there is a need to distinguish or indicate different that the environment is somehow different (e.g. when dealing with multiple prompts like we were doing previously).